.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 /*.js
+!.eslintrc.js
```

Comment:
This will be fun when we go to remove all the built JS files with `yarn build`—can't just `rm *.js` anymore. :sigh:

Reply:
:sob: